src/router/routerConfig.js

```diff
@@ -0,0 +1,41 @@
+import HomePage from 'pages/HomePage.jsx';
+import RegisterPage from 'pages/RegisterPage.jsx';
+import LoginPage from 'pages/LoginPage.jsx';
+import MenuPage from 'pages/MenuPage.jsx';
+import CartPage from 'pages/CartPage.jsx';
+import ErrorPage from 'pages/ErrorPage.jsx';
+
+const routes = [
+    {
+        path: '/',
+        element: HomePage,
+        protected: undefined,
+    },
+    {
+        path: '/login',
+        element: LoginPage,
+        protected: 'guest',
```

Comment:
I would suggest checking if user iside of login page and leave only 2 access options protected and public